packages/sdk-cli/src/models/DeveloperRelay.ts

```diff
@@ -0,0 +1,107 @@
+import stream from 'stream';
+import { promises as fsPromises } from 'fs';
+
+import * as t from 'io-ts';
+import websocketStream from 'websocket-stream';
+
+import * as localRelay from './localRelay';
+import { apiFetch, assertAPIResponseOK, decodeJSON } from '../api/baseAPI';
+import { assertContentType } from '../util/fetchUtil';
+import environment from '../auth/environment';
+
+// tslint:disable-next-line:variable-name
+export const Host = t.type(
+  {
+    id: t.string,
+    displayName: t.string,
+    roles: t.array(t.string),
+    state: t.union([t.literal('available'), t.literal('busy')]),
+  },
+  'Host',
+);
+export type Host = t.TypeOf<typeof Host>;
+
+export type Hosts = { appHost: Host[]; companionHost: Host[] };
+
+// tslint:disable-next-line:variable-name
+const HostsResponse = t.type(
+  {
+    hosts: t.array(Host),
+  },
+  'HostsResponse',
+);
+
+export default class DeveloperRelay {
+  constructor(
+    private readonly apiUrl: string = environment().config.apiUrl,
+    private readonly shouldAuth: boolean = true,
+  ) {}
+
+  static async create(local = false) {
+    if (local) {
+      // Create an output directory (if doesn't exist)
+      try {
+        await fsPromises.mkdir(localRelay.RELAY_DIRECTORY_PATH);
+      } catch (error) {
+        if ((error as NodeJS.ErrnoException).code !== 'EEXIST') {
+          throw new Error('Error creating a directory for Local Relay files');
```

Comment:
Is it actually an error if the directory already exists?

Reply:
Just checked it through node repl, it is indeed. I think with `option.recursive` it won't throw an error but afaik it has some edge cases which will be too much effort to deal with

```
 [Error: EEXIST: file already exists, mkdir '/Users'] {
  errno: -17,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: '/Users'
}
```

Reply:
Are we doing any directory cleanup somewhere or the directory needs to be deleted manually after each run?

Reply:
Sorry, accidentally resolved. The directory is supposed to contain logs and a PID file, the PID file _should_ be cleaned up automatically by the local-relay process (if the process fails to do so, say in case of a crash, the PID file in the dir needs to be deleted manually), but the logs should remain. So no, the directory doesn't need cleanup or manual deletion

Reply:
Resolving this as I see that this part of the code was removed.